metagenomics_samplesheet_generator.ipynb

```diff
@@ -896,7 +880,9 @@
     "         'BarcodesAreRC': 'False',\n",
     "         'ForwardAdapter': 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC',\n",
     "         'ReverseAdapter': 'GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT',\n",
-    "         'HumanFiltering': 'True'\n",
+    "         'HumanFiltering': 'True',\n",
+    "         'library_construction_protocol': '',\n",
+    "         'experiment_design_description': '',\n",
```

Comment:
Same as with other experiment design descriptions. Might be easier to handle this in the validator.